account_skip_bank_reconciliation/models/account_reconcile_model.py

```diff
@@ -1,16 +1,14 @@
 # Copyright 2020 ForgeFlow S.L.
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, models
+from odoo import models
 
 
 class AccountReconcileModel(models.Model):
     _inherit = "account.reconcile.model"
 
-    @api.multi
     def _apply_conditions(self, query, params):
-        query, params = super(AccountReconcileModel, self)._apply_conditions(
-            query, params
-        )
-        query += " AND account.exclude_bank_reconcile IS NOT TRUE"
+        query, params = super()._apply_conditions(query, params)
+        if "account_account account" in query:
```

Comment:
Maybe you could cherry-pick https://github.com/OCA/account-reconcile/commit/34fc6be367fac77d856b59c9f29571bdc80e9ad8 instead, which is more readable :octopus:

Reply:
adapted to your suggestion :)